src/instructlab/utils.py

```diff
@@ -28,15 +28,21 @@
     TaxonomyParser,
     TaxonomyReadingException,
 )
+from instructlab.training.chat_templates import (
+    ibm_legacy_tmpl as granite_legacy,  # type: ignore
+)
 import click
 import git
 import gitdb
 import yaml
 
 # Local
 from . import common
+from .common import CLI_HELPER_SYS_PROMPT, SYSTEM_PROMPTS, SupportedModelArchitectures
 from .defaults import DEFAULTS
 
+# mypy: disable_error_code="import-untyped"
```

Comment:
same here, I think there is a pylint version of this too, right?

Reply:
probably? not sure
I just did the mypy one to get around the linter